docs/installing.md

```diff
@@ -1,133 +1,26 @@
-
-The commands below will install Tracetest connecting to the Jaeger tracing backend on `jaeger-query.observability.svc.cluster.local:16685`.
-
-```sh
-# Install Kubeshop Helm repo and update it
-helm repo add kubeshop https://kubeshop.github.io/helm-charts
-helm repo update
-
-helm install tracetest kubeshop/tracetest \
-  --set telemetry.dataStores.jaeger.jaeger.endpoint="jaeger-query.observability.svc.cluster.local:16685" `# update this value to point to your jaeger install` \
-  --set telemetry.exporters.collector.exporter.collector.endpoint="otel-collector.tracetest.svc.cluster.local:4317" `# update this value to point to your collector install` \
-  --set server.telemetry.dataStore="jaeger"
 ```
+$ curl -L https://raw.githubusercontent.com/kubeshop/tracetest/main/install-cli.sh | sh
+$ tracetest server install
 
-#### **Grafana Tempo**
-
-Tracetest uses [Grafana Tempo's Server's `9095` port](https://grafana.com/docs/tempo/latest/configuration/#server) to find Traces using gRPC protocol.
-
-The commands below will install the Tracetest application connecting to the Grafana Tempo tracing backend on `grafana-tempo:9095`:
-
-```sh
-# Install Kubeshop Helm repo and update it
-helm repo add kubeshop https://kubeshop.github.io/helm-charts
-helm repo update
-
-helm install tracetest kubeshop/tracetest \
-  --set telemetry.dataStores.tempo.tempo.endpoint="grafana-tempo:9095" \ # update this value to point to your tempo install
-  --set telemetry.exporters.collector.exporter.collector.endpoint="otel-collector:4317" \ # update this value to point to your collector install
-  --set server.telemetry.dataStore="tempo"
+# Follow the install wizard
 ```
 
-#### **Opensearch**
+## How it works
 
-The commands below will install the Tracetest application connecting to the Opensearchtracing backend on `opensearch:9200`:
-
-```sh
-# Install Kubeshop Helm repo and update it
-helm repo add kubeshop https://kubeshop.github.io/helm-charts
-helm repo update
-
-helm install tracetest kubeshop/tracetest \
-  --set telemetry.dataStores.opensearch.opensearch.addresses={"http://opensearch:9200"} \ # update this value to point to your opensearch install
-  --set telemetry.dataStores.opensearch.opensearch.index="traces" \ # update this value to use the index where your traces are being stored
-  --set telemetry.dataStores.opensearch.opensearch.username="admin" \ # update this value with your opensearch username
-  --set telemetry.dataStores.opensearch.opensearch.password="admin" \ # update this value with your opensearch password
-  --set server.telemetry.dataStore="opensearch"
-```
+Tracetest is a relatively complex system, with a few intercontected dependencies. It relies on a Postgres database, a Data Store to retrive your application's traces,
+on the OpenTelemetry Collector connected to the Data Store, etc. If you are new to the tracing world, it might be overwhelming, and a bit tricky to connect everything togheter.
```

Comment:
```suggestion
on the OpenTelemetry Collector connected to the Data Store, etc. If you are new to the tracing world, it might be overwhelming and a bit tricky to connect everything together.
```